### PR TITLE
Add getters for EventMotion properties.

### PR DESCRIPTION
**Pull request by hmeyer**
also add myself and my employer to AUTHORS.

### CODE REVIEW

File: src/event_motion.rs [hunk 1]
```diff
@@ -3,9 +3,24 @@
...
 event_wrapper!(EventMotion, GdkEventMotion);
 event_subtype!(EventMotion, MotionNotify);
+
+impl EventMotion {
+    pub fn get_position(self: Self) -> (f64, f64) {
+        let x = self.as_ref().x;
+        let y = self.as_ref().y;
+        (x, y)
+    }
+    pub fn get_state(self: Self) -> ffi::GdkModifierType {
```
Comment (GuillaumeGomez) [inline src/event_motion.rs]: missing empty line

File: src/event_motion.rs [hunk 2]
```diff
@@ -3,9 +3,24 @@
...
+        self.as_ref().state
+    }
+    pub fn get_time(self: Self) -> u32 {
```
Comment (GuillaumeGomez) [inline src/event_motion.rs]: missing empty line

File: COPYRIGHT [hunk 3]
```diff
@@ -17,8 +17,10 @@ Evgenii Pashkin <eapashkin@gmail.com>
 Geoffrey French <frondit1985@gmail.com>
 Gleb Kozyrev <gleb@gkoz.com>
 Glenn Watson <gw@intuitionlibrary.com>
+Google Inc.
```
Comment (GuillaumeGomez) [inline COPYRIGHT]: I'm feeling uneasy with this...

**Comment by gkoz** [comment]
@hmeyer Could you please elaborate about the role of your employer here? It doesn't seem likely that two parties at once hold copyright for this patch.

On the technical side, the getters should borrow `self` (e.g. `fn get_time(&self) -> u32`) not consume it. We try to avoid using `ffi` type names in the higher level APIs, you can use it under the `::ModifierType` name instead. The `use gdk_ffi as ffi;` line is not needed after #103.

**Comment by gkoz** [comment]
> The use gdk_ffi as ffi; line is not needed after #103.

Or rather, this needs a rebase.

**Comment by hmeyer** [comment]
Disclaimer: I am not a lawyer.
But as I understand it, my employer holds the copyright for all my software development work. So that name needs to go into COPYRIGHT. I consequently removed my name.
Also I addressed the other comments.

**Comment by hmeyer** [comment]
Google employees participate in many open source projects, and Google Inc is consequently added to AUTHORS/COPYRIGHT. For example:
http://hg.mozilla.org/comm-central/file/tip/AUTHORS#l170

**Comment by gkoz** [comment]
> as I understand it, my employer holds the copyright for all my software development work.

I'm sad to hear this. If you don't own the code there's so far no reason for us to believe it has been open-sourced by the owner.

**Comment by GuillaumeGomez** [comment]
@hmeyer: Can you squash your commits please?

File: src/event_motion.rs [hunk 4]
```diff
@@ -3,9 +3,26 @@
...
+use ffi::GdkModifierType;
 
 #[derive(Clone, Debug)]
 pub struct EventMotion(::Event);
 
 event_wrapper!(EventMotion, GdkEventMotion);
 event_subtype!(EventMotion, MotionNotify);
+
+impl EventMotion {
+    pub fn get_position(self: &Self) -> (f64, f64) {
```
Comment (gkoz) [inline src/event_motion.rs]: The convention is `&self` not `self: &Self`.

File: src/event_motion.rs [hunk 5]
```diff
@@ -3,9 +3,26 @@
...
+        let x = self.as_ref().x;
+        let y = self.as_ref().y;
+        (x, y)
+    }
+
+    pub fn get_state(self: &Self) -> GdkModifierType {
```
Comment (gkoz) [inline src/event_motion.rs]: Maybe I wasn't clear, I want `ModifierType` here, it's reexported in this crate (see `lib.rs`).

**Comment by hmeyer** [comment]
Addressed the comments (and learned how to squash commits).
COPYRIGHT: There is a LICENSE file in the project which Google Inc. is aware of and to which it agrees. Think about this: If I had just added myself to COPRYRIGHT - what reason would you have to believe I have open-sourced m the patch?

**Comment by gkoz** [comment]
I think your employer has a policy and process for such cases designed to eliminate uncertainty. Saying that Google agrees to the license comes off as skirting that process.

**Comment by hmeyer** [comment]
Yes, there is a process. And yes, I went through that process and got approval.

**Comment by gkoz** [comment]
Great! Does that process allow you to paste here a copy of the approval or is it confidential?

**Comment by hmeyer** [comment]
The approval was: "LGTM"

**Comment by gkoz** [comment]
I guess that will do. Thanks!

**Comment by GuillaumeGomez** [comment]
Thanks a lot for your work @hmeyer!

**Pull request merged**